assignments/prototypes.js

```diff
@@ -32,16 +57,78 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+
+function Humanoid (character) {
+  this.team = character.team;
+  this.weapons = character.weapons;
+  this.language = character.language;
+  CharacterStats.call(this, character);
+}
+
+Humanoid.prototype = Object.create(CharacterStats.prototype);
+
+Humanoid.prototype.greet = function () {
+  return `${this.name} offers a greeting in ${this.language}.`
+}
+
+
 /*
   * Inheritance chain: GameObject -> CharacterStats -> Humanoid
   * Instances of Humanoid should have all of the same properties as CharacterStats and GameObject.
   * Instances of CharacterStats should have all of the same properties as GameObject.
 */
 
-// Test you work by un-commenting these 3 objects and the list of console logs below:
 
-/*
+//VILLAIN ------------------------
+
+function Villain (character) {
+  Humanoid.call(this, character)
```

Comment:
Great job finishing up the stretch goals! (Even if you were a whole minute late to the standup meeting)